### PR TITLE
ci: Add Echidna assertion test mode

### PR DESCRIPTION
**Pull request by StefanVitor**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -184,33 +184,58 @@ jobs:
...
+          pkg-manager: yarn
+      - run:
+          name: "Run Echidna assertion lido pool"
+          no_output_timeout: 30m
+          command: |
+            yarn echidna-test-assertion:pool:lido
+
+  echidna-assertion-yearn-pool:
+    docker:
+      - image: stefantempus/tempus-echidna:latest
```
Comment (mijovic) [inline .circleci/config.yml:228]: Would be nice to have tempus/echidna

**Review by mijovic — COMMENTED** [review]
Looks good now with docker optimization

File: .circleci/config.yml (hunk 1 above)
Comment (StefanVitor) [reply .circleci/config.yml:228]: I wanted that and initially, but I misread that only paid users could create "organization" on docker hub. However, it's possible that and the free users create an organization with three members, so I now created 'tempusfinance' (only 'tempus` is occupied). We will see how this with docker hub works in next period, and maybe we will have to upgrade on payed version (it's about 300$ per year for 5 user).

File: .circleci/config.yml [hunk 2]
```diff
@@ -184,33 +184,58 @@ jobs:
...
-          name: "Run Echidna"
+          name: "Run Echidna yearn pool"
           no_output_timeout: 30m
           command: |
-            yarn echidna-test:pool:lido
             yarn echidna-test:pool:yearn
 
+  echidna-assertion-lido-pool:
+    docker:
+      - image: stefantempus/tempus-echidna:latest
```
Comment (mijovic) [inline .circleci/config.yml:215]: Here it is still the same

File: .circleci/config.yml [hunk 3]
```diff
@@ -184,33 +184,58 @@ jobs:
...
+          pkg-manager: yarn
+      - run:
+          name: "Run Echidna assertion lido pool"
+          no_output_timeout: 30m
+          command: |
+            yarn echidna-test-assertion:pool:lido
+
+  echidna-assertion-yearn-pool:
+    docker:
+      - image: stefantempus/tempus-echidna:latest
```
Comment (mijovic) [inline .circleci/config.yml:228]: And here as well

Comment (StefanVitor) [reply .circleci/config.yml]: Oops, forgot those two, I also rebased after the previous merge, so I think that now everything is fine.

**Review by mijovic — APPROVED** [review]

**Pull request merged**